### DISCUSSION TITLE
Ability to Cast to a Cast-Enabled Device

### DISCUSSION THREAD

**Discussion by Anon666333** [immich-app/immich#14488] [Feature Request]
In Google Photos, there's an option at the top of any photo to cast it to a cast-enabled device. This is great for sharing photos with a group around a TV, instead of sharing photos or passing a phone around. 

Is this something that would be feasible to add to immich?

Thank you so much for all the hard work on this 🙏🙏

**Comment by bo0tzz** [comment]
#1614
